package.json

```diff
@@ -33,6 +33,7 @@
   "author": "OpenFGA",
   "dependencies": {
     "@openfga/sdk": "^0.2.7",
+    "@openfga/language": "./src/language/pkg/js",
```

Comment:
Curious why we do this - if we intend to have language as part of src, can we not just import them without specifying it as dependencies here?

Reply:
Yep, thats fine. I just thought the change could be made a little more seamlessly for here: https://github.com/openfga/syntax-transformer/pull/152/files#diff-a2a171449d862fe29692ce031981047d7ab755ae7f84c707aef80701b3ea0c80R10

But happy to take it out.

Reply:
After change, rediscovered this was an easy way of not having to declare a top level dependency on `antlr`